crypto/cryptlib.c

```diff
@@ -272,6 +289,17 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_end(ap);
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#ifdef OPENSSL_SYS_WIN_CORE
+    /* ONECORE is always NONGUI and NT >= 0x0601*/
+        HRESULT tEventLog = TraceLoggingRegister(g_hProvider);
+        if (SUCCEEDED(tEventLog)) {
+            const TCHAR *pmsg = buf;
```

Comment:
Why this assignment?

Reply:
Based on similar assignment at line 310, unnecessary